### PR TITLE
fix(rbac): update the RBAC frontend plugin readme

### PR DESCRIPTION
**Pull request by debsmita1**
**Resolves:**
https://github.com/janus-idp/backstage-plugins/issues/1114

### CODE REVIEW

File: plugins/rbac/README.md [hunk 1]
```diff
@@ -21,14 +21,28 @@ Follow the RBAC backend plugin [README](https://github.com/janus-idp/backstage-p
...
+permission:
+  enabled: true
+  rbac:
+    pluginsWithPermission:
+      - kubernetes
+      - jenkins
+    admin:
+      users:
+        - name: user:default/<user-name>
+```
```
Comment (invincibleJai) [inline plugins/rbac/README.md:44]: What happens if the user adds this in the config explicitly?

File: plugins/rbac/README.md [hunk 2]
```diff
@@ -21,14 +21,28 @@ Follow the RBAC backend plugin [README](https://github.com/janus-idp/backstage-p
...
 **NOTE**
 
-To enable create role button on Administration -> RBAC roles list page, the role associated with your user should have the following permission policies associated with it. Add the following in your permission policies configuration file:
+- To enable create role button on Administration -> RBAC roles list page, the role associated with your user should have the following permission policies associated with it. Add the following in your permission policies configuration file:
 
 ```CSV
-p, role:default/team_a, catalog-entity, read, allow
+p, role:default/team_a, catalog-entity, read, allow  # only needed for users with the role other than `role:default/rbac_admin`
 p, role:default/team_a, policy-entity, create, allow
 g, user:default/<login-id/user-name>, role:default/team_a
```
Comment (invincibleJai) [inline plugins/rbac/README.md:29]: this looks a bit confusing based on Readme if you add a user in config https://github.com/janus-idp/backstage-plugins/pull/1155/files#diff-8ede316b0bcf371c5e1b99eebe533e18667172539b0492c668976d14af9e9483R43 then they automatically will be mapped to `role:default/rbac_admin` and they will have access to so should we have this note?

https://github.com/janus-idp/backstage-plugins/pull/1137 handled this

@AndrienkoAleksandr WDYT?

File: plugins/rbac/README.md [hunk 3]
```diff
@@ -21,14 +21,28 @@ Follow the RBAC backend plugin [README](https://github.com/janus-idp/backstage-p
...
 
+- To view the permissions from other plugins like `Kubernetes` and `Jenkins` as mentioned [here](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/rbac-backend/docs/permissions.md), add the following configuration in your `app-config.yaml`:
+
+```yaml title="app-config.yaml"
+permission:
+  enabled: true
+  rbac:
+    pluginsWithPermission:
+      - kubernetes
+      - jenkins
```
Comment (invincibleJai) [inline plugins/rbac/README.md:40]: What happens if the user adds this in the config explicitly?

File: plugins/rbac/README.md (hunk 2 above)
Comment (debsmita1) [reply plugins/rbac/README.md:29]: I have updated this to mention non-admin users

File: plugins/rbac/README.md (hunk 3 above)
Comment (debsmita1) [reply plugins/rbac/README.md:40]: @invincibleJai  Sharing some screenshots here: 

Screenshot 2024-02-02 at 12 38 08 PM
Screenshot 2024-02-02 at 12 38 59 PM

**Review by AndrienkoAleksandr — APPROVED** [review]

**Pull request merged**